### PR TITLE
Major revision to model format_specs

### PR DESCRIPTION
**Pull request by WesIngwersen**

### CODE REVIEW

**Comment by WesIngwersen** [comment]
I'm not sure why Rho and Phi are dataframes and not matrices.  @MoLi7

**Comment by WesIngwersen** [comment]
`FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`.  @MoLi7 @jvendries @bl-young  thoughts?

**Comment by WesIngwersen** [comment]
Right now I have the following categories present in items in the model table:
1. metadata
2. supporting data
3. component matrices
4. component data
5. result matrices

We have used component and result matrices in the documentation.  I'm up for discussion on the others.

**Comment by WesIngwersen** [comment]
The latest commits were a big refactor of storage of the config files to move them into a subfolders to make it much easier to find them and separate out the model and disagg config files from the rest of the extdata files

File: R/DisaggregateFunctions.R [hunk 1]
```diff
@@ -6,7 +6,7 @@ disaggregateModel <- function (model){
   for (disagg in model$specs$DisaggregationSpecs){
     disaggregationConfigFile <- disagg
     logging::loginfo(paste("Reading disaggregation for", disaggregationConfigFile, sep=" "))
-    model$DisaggregationSpecs <- getModelConfiguration(disaggregationConfigFile)
+    model$DisaggregationSpecs <- getConfiguration(disaggregationConfigFile, "disagg")
```
Comment (bl-young) [inline R/DisaggregateFunctions.R:9]: The other disagg files (e.g. WasteDisaggregation_Specs, WasteDisaggregationDetail_Make, etc.) could also be moved to that disaggspecs folder with a minor change to lines 15,29,33. and 36

**Comment by MoLi7** [comment]
> I'm not sure why `Rho` and `Phi` are dataframes and not matrices. @MoLi7

`Rho` and `Phi` both stemmed from and have the same rows and columns with `model$MultiyearCPI` (which is a `data.frame`), so they remained `data.frame`. But I'm okay with converting `Rho` and `Phi` to `matrix`. @WesIngwersen

**Comment by MoLi7** [comment]
> `FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`. @MoLi7 @jvendries @bl-young thoughts?

I like `Components` but am open to other options. Whichever name we decide to use should also be applied to `MarginSectors`. @WesIngwersen

**Comment by WesIngwersen** [comment]
> > `FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`. @MoLi7 @jvendries @bl-young thoughts?
> 
> I like `Components` but am open to other options. Whichever name we decide to use should also be applied to `MarginSectors`. @WesIngwersen

OK good.  But I think the Margins are actually sectors as we define them. Of course we can rename this object just to make it as clear as possible.

**Comment by WesIngwersen** [comment]
> > `FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`. @MoLi7 @jvendries @bl-young thoughts?
> 
> I like `Components` but am open to other options. Whichever name we decide to use should also be applied to `MarginSectors`. @WesIngwersen

Decided to go with "Meta"

File: R/DisaggregateFunctions.R (hunk 1 above)
Comment (bl-young) [reply R/DisaggregateFunctions.R:9]: This has been updated with https://github.com/USEPA/useeior/pull/140/commits/86f3b054adf5cc41dfdcfdc955d5c83969afac49

**Comment by MoLi7** [comment]
> >  `FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`.
> 
> Decided to go with "Meta"

@WesIngwersen I thought you'd want to change `FinalDemandSectors` to `FinalDemandMeta` (same for `ValueAddedSectors`) throughout the scripts, do you want me to apply the changes?

**Comment by WesIngwersen** [comment]
> I'm not sure why `Rho` and `Phi` are dataframes and not matrices. @MoLi7



> > > `FinalDemandSectors` and `ValueAddedSectors` have `Sectors` when they are not sectors. I'm considering other names like `Components` or `Categories`.
> > 
> > 
> > Decided to go with "Meta"
> 
> @WesIngwersen I thought you'd want to change `FinalDemandSectors` to `FinalDemandMeta` (same for `ValueAddedSectors`) throughout the scripts, do you want me to apply the changes?

I did it, thanks

**Comment by WesIngwersen** [comment]
FYI @MoLi7 @bl-young merging this

**Pull request merged**